packages/utils/src/constants.ts

```diff
@@ -16,6 +16,7 @@ export const ERRORS_KEY = '__errors';
 export const ID_KEY = '$id';
 export const IF_KEY = 'if';
 export const ITEMS_KEY = 'items';
+export const JUNK_OPTION_ID = 'junk_option_schema_id';
```

Comment:
open to suggestions for the junk option id constant

Reply:
Let's see if this works ok when compiling.
```suggestion
export const JUNK_OPTION_ID = '_$junk_option_schema_id$_';
```